scraper/src/youtube2zim/entrypoint.py

```diff
@@ -86,6 +87,15 @@ def main():
         default=False,
     )
 
+    parser.add_argument(
+        "--zimui-dist",
+        type=str,
+        help=(
+            "Directory containing Vite build output from the Zim UI Vue.JS application"
+        ),
+        default=os.getenv("KOLIBRI_ZIMUI_DIST", "../zimui/dist"),
```

Comment:
`KOLIBRI_ZIMUI_DIST` is probably not the right environment variable name ;)

Reply:
Oops 😅 Fixed it in [304e7c3](https://github.com/openzim/youtube/pull/207/commits/304e7c367e9ff184ea5865ae1499f880115766d4)